addon/mixins/route.js

```diff
@@ -199,6 +200,17 @@ export default Ember.Mixin.create({
   },
 
   /**
+   Call additional functions after finding the infinityModel in the Ember data store.
+
+   @method afterInfinityModel
+   @param {Function} infinityModelPromise The resolved result of the Ember store find method. Passed in automatically.
+   @return {Ember.RSVP.Promise}
+   */
+  afterInfinityModel(infinityModelPromiseResult) {
+    return infinityModelPromiseResult;
```

Comment:
since this gets used as a chained promise, it requires the end-user code to make sure to call `_super` and also return the correct result needed downstream by `updateInfinityModel`. It would be better if it was an optional hook like so:

```javascript
_afterInfinityModel(infinityModelPromiseResult) {
  if (typeof this.afterInfinityModel === 'function') {
    this.afterInfinityModel(infinityModelPromiseResult);
  }
  return infinityModelPromiseResult;
},
```